locator/tablets.hh

```diff
@@ -348,6 +348,9 @@ public:
     /// \throws std::logic_error If the given id does not belong to this instance.
     dht::token_range get_token_range(tablet_id id) const;
 
+    /// Returns a vector of contiguous tokens where each pair describes a range of tablet.
+    seastar::future<std::vector<token>> get_flat_token_ranges() const;
+
```

Comment:
seastar namespace is used in most of places, so no need to specify it.

Reply:
Thanks, fixed!